ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +62,38 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_disks_by_path():
+    oc_obj = OCP()
+    disks = oc_obj.exec_oc_debug_cmd(
+        node=constants.SNO_NODE_NAME, cmd_list=["ls /dev/disk/by-path"]
+    )
+    raw_disks_list = disks.split("\n")
+    raw_disks_list = list(filter(None, raw_disks_list))
+    disks_by_path = list()
+    for line in raw_disks_list[-storage_disks_count:]:
+        disk_name = "/dev/disk/by-path/" + line
+        disks_by_path.append(disk_name)
+
+    return disks_by_path
+
+
+def get_disks_by_name():
```

Comment:
I would suggest renaming of this function to `get_block_devices` as that is what lsblk does